src/ceph-disk/ceph_disk/main.py

```diff
@@ -1813,6 +1813,43 @@ def set_subparser(subparsers):
         parser = subparsers.add_parser(
             'prepare',
             parents=parents,
+            formatter_class=argparse.RawDescriptionHelpFormatter,
+            description=textwrap.fill(textwrap.dedent("""\
+            If the --bluestore argument is given, a bluestore objectstore
+            will be used instead of the legacy filestore objectstore.
+
+            When an entire device is prepared for bluestore, two
+            partitions are created. The first partition is for metadata,
+            the second partition is for blocks that contain data.
+
+            Unless explicitly specified with --block.db or
+            --block.wal, the bluestore DB and WAL are files within the
+            the metadata directory. For instance:
```

Comment:
Oops, actually:

Unless explicitly specified with --block.db or --block.wal, the bluestore DB and WAL data is stored on the main block device.